### PR TITLE
Contribution guidelines

### PR DESCRIPTION
**Pull request by sgratiy**
Relates to: #409

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Change Log
```
Comment (NileGraddis) [inline CHANGELOG.md:1]: decided not to go with github releases?

Comment (sgratiy) [reply CHANGELOG.md:1]: No I did not. Should we not use CHANGELOG.md if we use github release? How does it work?

**Comment by sgratiy** [comment]
@MatthewAitken could you please elaborate on you point
>little bit more info on how that works to the release process

File: CHANGELOG.md (hunk 1 above)
Comment (NileGraddis) [reply CHANGELOG.md:1]: well I guess we don't need a markdown changelog if we are going with github. Or we could track  "latest changes" in the repo while the full log is on the releases (as in allensdk).

Comment (sgratiy) [reply CHANGELOG.md:1]: I have updated the guidelines, stating that release notes are pulled from the changelog which in turn must be updated by committers.

**Pull request merged**